widgets/userGroups/src/CreateModal.tsx

```diff
@@ -100,15 +83,14 @@ const CreateModal = ({ toolbox, isLdapEnabled = false }: CreateModalProps) => {
             .finally(unsetLoading);
     }
 
-    function handleRoleChange(tenant: string, role: Role) {
-        const newTenants: INewTenants = { ...tenants };
-        newTenants[tenant] = role;
+    function handleRoleChange(tenant: string, role: string | undefined) {
+        const newTenants: NewTenants = { ...tenants, [tenant]: role };
         setTenants(newTenants);
     }
 
     function handleTenantChange(_proxy: any, field: { value?: any }) {
-        const newTenants: INewTenants = {};
-        _.forEach(field.value, tenant => {
+        const newTenants: NewTenants = {};
+        _.map(field.value, tenant => {
             newTenants[tenant] =
                 tenants[tenant] || Stage.Common.Roles.Utils.getDefaultRoleName(toolbox.getManagerState().roles);
         });
```

Comment:
How about:
```suggestion
        const newTenants = _(field.value).keyBy().mapValues(tenant => tenants[tenant] || Stage.Common.Roles.Utils.getDefaultRoleName(toolbox.getManagerState().roles));
```
or maybe:
```suggestion
        const newTenants = _(field.value).keyBy().mapValues(tenant => tenants[tenant] || Stage.Common.Roles.Utils.getDefaultRoleName(toolbox.getManagerState().roles));
```

Reply:
Yes, sorry about that, I edited the comment.

Reply:
after applying these changes I'm getting this result: ![Zrzut ekranu 2022-06-21 120207](https://user-images.githubusercontent.com/17828621/174774589-52822c90-bbf3-4605-a839-6e4581c961de.jpg) 🤔

Reply:
Looks like a call to `value()` is needed at the end of the chain. I'll update the suggestion.